.github/workflows/publishbot.yml

```diff
@@ -0,0 +1,46 @@
+name: Publish Robot
+on:
+  push:
+    branches:
+      - master
+    tags: ['*']
+jobs:
+  build:
+    name: "Build"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - uses: ./.github/actions/setup
+  npm-publish:
+    name: 'Publish to NPM'
+    needs: build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - uses: ./.github/actions/setup
+      - run: npm run npm:publish:alpha
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+  deploy:
+    name: 'Deploy the package'
+    needs: build
+    runs-on: [self-hosted, linux, x64, ec2-alpha]
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - uses: ./.github/actions/setup
+      - run: rm -rf veracode && mkdir veracode
+      - run: mkdir veracode/auth
+      - run: cp -R packages/auth/src packages/auth/package.json packages/auth/package-lock.json veracode/auth
+      - run: mkdir veracode/bueno
+      - run: cp -R packages/bueno/src packages/bueno/package.json packages/bueno/package-lock.json veracode/bueno
+      - run: mkdir veracode/headless
+      - run: cp -R packages/headless/src packages/headless/package.json packages/headless/package-lock.json veracode/headless
+      - run: mkdir veracode/atomic
+      - run: cp -R packages/atomic/src packages/atomic/package.json packages/atomic/package-lock.json veracode/atomic
```

Comment:
We could:
* create the veracode package here (current solution)
* create the veracode package in a dedicated action
* create the veracode package in `create-deployment-package`

I don't feel strongly about any of those.

Reply:
Indeed, there are plans to mode to Snyk code next year, but I don't think the deployment pipeline has a certifier for it yet. I think the current solution looks good in the interim,

Reply:
if we switch to Snyk Code, I'd prefer to do it in a separate PR. I'll ask about it.